File: trunk/Projects/Framework-Samples/NHibernate/CSharp/Sample.Data/ManagerObjects/AccountManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using Sample.Data.Generated.BusinessObjects;
using Sample.Data.Generated.Base;

namespace Sample.Data.Generated.ManagerObjects
{
    public partial interface IAccountManager : IManagerBase<Account, int>
    {
	}
	
	partial class AccountManager : ManagerBase<Account, int>, IAccountManager
    {
	}
}

File: Framework-Samples/NHibernate/VisualBasic/Sample.Data/ManagerObjects/AccountManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports Sample.Data.Generated.BusinessObjects
Imports Sample.Data.Generated.Base

Namespace Sample.Data.Generated.ManagerObjects
	'Public Partial Interface IAccountManager
	'	Inherits IManagerBase(Of Account, System.Int32)
	'End Interface

	Partial Class AccountManager
		Inherits ManagerBase(Of Account, System.Int32)
		Implements IAccountManager
	End Class
End Namespace
